timeslots/handler.go

```diff
@@ -0,0 +1,51 @@
+package timeslots
+
+import (
+	"encoding/json"
+	"net/http"
+	"timezone-converter/db"
+)
+
+type TimeslotData struct {
+	UserId string `json:"userId"`
+}
+
+func CreateTimeslots(w http.ResponseWriter, r *http.Request) {
+	var timeslotsData TimeslotData
+	json.NewDecoder(r.Body).Decode(&timeslotsData)
+
+	repo := NewRepository(db.DbInstance)
+
+	time := [6]string{"10:00", "11:00", "12:00", "13:00", "14:00", "15:00"}
```

Comment:
todo:
potentially find something better than just having these types of strings for available times.

Reply:
```
parameter: [{from, to}, {from, to}, ...]
```